### PR TITLE
Update nwfilter with parameters

### PR DESCRIPTION
**Pull request by yalzhang**
Live update the nwfilter with parameters. Depends on https://github.com/avocado-framework/avocado-vt/pull/2598

Signed-off-by: yalzhang <yalzhang@redhat.com>

### CODE REVIEW

**Comment by yalzhang** [comment]
test locally pass except one case which fail because of one un-fixed bug 1661367 about delete rom
$ run virtual_network.iface_update
/root/code/avocado/avocado/plugins/run.py:299: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
JOB ID     : c87d9a8f1818cb3a5145b01201ae5e4023b7c894
JOB LOG    : /root/avocado/job-results/job-2020-06-18T10.31-c87d9a8/job.log
 (01/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.update_bandwidth: PASS (5.60 s)
 (02/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.update_link: PASS (30.56 s)
 (03/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.update_link_with_rom: PASS (30.72 s)
 (04/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.update_link_without_addr: PASS (31.71 s)
 (05/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.update_source: PASS (5.76 s)
 (06/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.add_filter: PASS (5.60 s)
 (07/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.update_filter: PASS (6.43 s)
 (08/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.update_filter_parameter: PASS (7.56 s)
 (09/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.del_filter: PASS (6.89 s)
 (10/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.cold_update_alias: PASS (4.13 s)
 (11/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.update_alias: PASS (5.20 s)
 (12/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.update_queues: PASS (5.79 s)
 (13/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.update_driver: PASS (5.04 s)
 (14/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.update_model: PASS (5.34 s)
 (15/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.update_rom: PASS (4.97 s)
 (16/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.add_rom_enable: PASS (5.05 s)
 (17/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.update_rom_enable: PASS (5.36 s)
 (18/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.update_rom_and_address: PASS (5.44 s)
 (19/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.del_rom_enable: FAIL: Run '/usr/bin/virsh update-device avocado-vt-vm1 /var/tmp/xml_utils_temp_7q0_oooe.xml ' expect fail, but run successfully. (5.98 s)
 (20/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.update_bandwidth: PASS (5.03 s)
 (21/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.update_target: PASS (4.98 s)
 (22/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.update_address: PASS (5.03 s)
 (23/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.update_mtu.not_support: PASS (6.06 s)
 (24/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.update_mtu.invalid_number: PASS (5.98 s)
 (25/25) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.update_mtu.invalid_character: PASS (5.97 s)
RESULTS    : PASS 24 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 218.61 s

**Comment by yalzhang** [comment]
no change for existing cases, only add one new case:
virtual_network.iface_update.positive_test.update_filter_parameter

File: libvirt/tests/src/virtual_network/iface_update.py [hunk 1]
```diff
@@ -184,6 +186,11 @@ def check_mtu():
                 else:
                     test.fail("Get filter %s is not equal to set %s"
                               % (iface_filter_value, new_iface_filter))
+            if new_iface_filter_parameters:
+                ebtables_outputs = str(process.run("ebtables -t nat -L", shell=True).stdout)
```
Comment (chunfuwen) [inline libvirt/tests/src/virtual_network/iface_update.py:190]: process.run().stdout_text can ensure one  text string return

Comment (yalzhang) [reply libvirt/tests/src/virtual_network/iface_update.py]: upadted and Thank you!

**Pull request merged**